vllm/v1/cudagraph_dispatcher.py

```diff
@@ -1,9 +1,16 @@
 # SPDX-License-Identifier: Apache-2.0
 # SPDX-FileCopyrightText: Copyright contributors to the vLLM project
+from math import ceil
 from typing import Optional
 
+from typing_extensions import TypeAlias
+
 from vllm.config import CUDAGraphMode, VllmConfig
 from vllm.forward_context import BatchDescriptor
+from vllm.logger import init_logger
+
+logger = init_logger(__name__)
+CUDAGraphKey: TypeAlias = tuple[int, bool]
```

Comment:
Can we make this a dataclass with utilities for easier and more understandable handling as well as more docs?